sympy/polys/matrices/domainmatrix.py

```diff
@@ -36,6 +36,26 @@ def from_rep(cls, ddm):
 
     @classmethod
     def from_list_sympy(cls, nrows, ncols, rows, **kwargs):
+        """Converts a list to Domain Matrix
+
+        Examples
+        ========
+        Define a Domain Matrix on the field of Integers
```

Comment:
There should be a blank line under the `====` line.

DomainMatrix should be all one word and actually should be a sphinx reference like `:py:class:~.DomainMatrix`.